js/hotgraphicView.js

```diff
@@ -42,7 +42,7 @@ define([
     }
 
     reRender() {
-      if (Adapt.device.screenSize === 'large') return;
+      if (Adapt.device.screenSize === 'large' || !this.model.get('_isNarrativeOnMobile')) return;
```

Comment:
```suggestion
      if (Adapt.device.screenSize === 'large' || this.model.get('_isNarrativeOnMobile') === false) return;
```
Otherwise existing Hot Graphics without explicit JSON configuration will change in functionality.